examples/smart-mastering-complete/README.md

```diff
@@ -1,7 +1,18 @@
 This project provides examples of various mastering features in DHF. 
 
-To try this project out, start with a clean MarkLogic instance and initialize this project via Gradle: 
-application via Gradle:
+## How to install
```

Comment:
Just FYI, this is some boilerplate that I'm about to add to the README for every DHF 5 project, in a separate PR.